### ISSUE TITLE
Audio occasionally stops working

### ISSUE THREAD

**Issue by ilanbiala** [toleda/audio_ALC892#3]
My ALC892 on OS X 10.9.3 occasionally stops outputting audio. There is no sound coming out of my speakers even though it says the correct output is selected. After a couple of minutes, the audio starts working again.

**Comment by toleda** [comment]
No, Issue specific to user system.  Digital audio does not have the problem.

**Comment by ilanbiala** [comment]
Digital meaning HDMI/optical audio? Most speakers aren't digital, so how can I get sound through a digital output?

**Comment by toleda** [comment]
Onboard codec/optical has the same problem. USB is digital.
